gecode/int/distinct/cbs.hpp

```diff
@@ -0,0 +1,202 @@
+/* -*- mode: C++; c-basic-offset: 2; indent-tabs-mode: nil -*- */
+/*
+ *  Main authors:
+ *     Samuel Gagnon <samuel.gagnon92@gmail.com>
+ *
+ *  Copyright:
+ *     Samuel Gagnon, 2018
```

Comment:
Do I put you in the copyright?

Reply:
No, of course not unless you copied stuff written by me...